.github/workflows/releases.yaml

```diff
@@ -436,3 +436,79 @@ jobs:
         working-directory: editors/vscode
         env:
           OPEN_VSX_TOKEN: ${{ secrets.OPEN_VSX_TOKEN }}
+
+  pypi_build_taplo_cli:
+    name: Build python wheels
+    needs: ["wait_for_ci"]
+    runs-on: ${{ matrix.os }}-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu, windows, macos]
+        arch: [x86, x64, aarch64]
+        exclude:
+          - os: windows
+            arch: aarch64
+          - os: ubuntu
+            arch: aarch64
+          - os: macos
+            arch: x86
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build sdist
+        if: matrix.os == 'ubuntu' && matrix.arch == 'x86_64'
```

Comment:
todo: fix

Reply:
`sdist` is just the source code, so no need to compile that on different architectures. That's why it was in a separate job. It is not really required though.
https://docs.python.org/3.10/distutils/sourcedist.html

Reply:
Yes, but the arch comparison was wrong and I was too tired to fix it yesterday